README.md

```diff
@@ -1,6 +1,6 @@
-# u8timeseries: easy manipulation and forecasting
+# u8timeseries: easy manipulation and forecasting time series
 
-![u8timeseries](static/images/logo-72dpi.png "u8timeseries")
+![u8timeseries](https://github.com/unit8co/u8timeseries/raw/develop/static/images/logo-72dpi.png "u8timeseries")
```

Comment:
Sorry for offtop but since you change README can you also update Docker steps and point to our dockerhub image ;]?

Reply:
Sure

Reply:
scripts are already updated and README part has no reference to old images